src/libraries/System.Reflection.Metadata/tests/System.Reflection.Metadata.Tests.csproj

```diff
@@ -129,6 +129,7 @@
     <None Include="Resources\Misc\Signed.cmd" />
     <None Include="Resources\Misc\Signed.cs" />
     <EmbeddedResource Include="Resources\Misc\Signed.exe" />
+    <WasmFilesToIncludeInFileSystem Include="$(AssemblyName).dll" TargetPath="%(Identity)" />
```

Comment:
maybe add the same comment about SRM opening itself as you added in tests.mobile.targets

Reply:
Sure, I'll wait to see if you can see a better way to fix the tests... if we decide to keep this, then I'll add a comment.